biscuit-auth/src/format/schema.proto

```diff
@@ -178,3 +181,39 @@ message ThirdPartyBlockContents {
   required bytes payload = 1;
   required ExternalSignature externalSignature = 2;
 }
+
+message AuthorizerSnapshot {
+  repeated string symbols = 1;
+  optional uint32 version = 2;
+  repeated SnapshotBlock blocks = 3;
+  required SnapshotBlock authorizerBlock = 4;
+  repeated GeneratedFacts generatedFacts = 5;
+  repeated Policy policies = 6;
+  repeated KeyMap publicKeyMap = 7;
+}
+
+message Origin {
+  oneof Content {
+    bool authorizer = 1;
```

Comment:
is there a protobuf equivalent of `()`? here having `authorizer` defined but equal to `false` would be an invalid state.

Reply:
(not a big deal anyway, since the current protobuf encoding already allows invalid states)

Reply:
fixed in 927761c, by introducing the `Empty` message